File: speedtest/xorspeed.c

int main()
{
  char target[476]="\00012312312091205iwejoweifjwiefjwoefjwoefjpowifjpowifjpoiejpiojwpiojgiojgiosjgepijgspijgpiojgpijgpiogjpijgpiosjgisjgpisjreijsrgojgposeijgposijgspiogjspoigjpoirgjirjgoigjpoijgpiorjg91";

  char xor[476]="woicjapowijcpowijcapoijcawoejcapoeijcpoijcpoijcoiwjcepoijcpoijcopwijcpowijcpioejcijcpioeji";
  int i;
  for(i=1;i<10000000;i++)
  {
    for(int j=0;j<476;j++)
    {	    
      target[j]^=xor[j];
    }
  }
  return(target[0]);
}

File: speedtest/xorspeed.py
target = bytearray(b'\x00' + b'12312312091205iwejoweifjwiefjwoefjwoefjpowifjpowifjpoiejpiojwpiojgiojgiosjgepijgspijgpiojgpijgpiogjpijgpiosjgisjgpisjreijsrgojgposeijgposijgspiogjspoigjpoirgjirjgoigjpoijgpiorjg91', 'latin-1')
xor = b'woicjapowijcpowijcapoijcawoejcapoeijcpoijcpoijcoiwjcepoijcpoijcopwijcpowijcpioejcijcpioeji'

for _ in range(10000000):
    target = bytearray(x ^ y for x, y in zip(target, xor))

print(chr(target[0]))
